CHANGELOG.md

```diff
@@ -6,6 +6,8 @@ Please add changes to "master", preferably ordered by their significance. (Most
 
 ### master
 
+- Fixed issue with interactive tutorials ([#30](https://github.com/lynxkite/lynxkite/issues/30)).
```

Comment:
Not strictly related to this PR, but I think the changelog should mention the other two bugfixes as well (use table as graph, create graph with python).

Reply:
> Not strictly related to this PR, but I think the changelog should mention the other two bugfixes as well (use table as graph, create graph with python).

Done, thanks!